utils/deployContract.ts

```diff
@@ -1,6 +1,6 @@
 const { cyan, displayResult } = require('./colors')
 
-async function deployContract(deploy, contract, deployer, args) {
+export async function deployContract(deploy: Function, contract: string, deployer: string, args: Array<any>) {
```

Comment:
If you import `import { DeploymentsExtension } from 'hardhat-deploy/types';` you should be able to write:
```suggestion
export async function deployContract(deploy: DeploymentsExtension['deploy'], contract: string, deployer: string, args: Array<any>) {
```